webpack.config.js

```diff
@@ -140,30 +163,43 @@ module.exports = {
 				from: `./packages/${ packageName }/build-style/*.css`,
 				to: `./build/${ packageName }/`,
 				flatten: true,
-				transform: ( content ) => {
-					if ( mode === 'production' ) {
-						return postcss( [
-							require( 'cssnano' )( {
-								preset: [
-									'default',
-									{
-										discardComments: {
-											removeAll: true,
-										},
-									},
-								],
-							} ),
-						] )
-							.process( content, {
-								from: 'src/app.css',
-								to: 'dest/app.css',
-							} )
-							.then( ( result ) => result.css );
-					}
-					return content;
-				},
+				transform: stylesTransform,
 			} ) )
 		),
+		new CopyWebpackPlugin( [
```

Comment:
Looks like there's repetition here, I wonder if it's possible to just merge this with the previous copy  plugin (by relaxing the "from"/"to"